problem-18/src/app/components/guard/admin.guard.ts

```diff
@@ -0,0 +1,10 @@
+import {CanActivateFn, Router} from '@angular/router';
+import {inject} from "@angular/core";
+
+export const adminGuard: CanActivateFn = (route, state) => {
```

Comment:
adminGuard is in the components folder, wrong folder structure

Reply:
moved to /app folder